src/components/checkout/templates/container/style.js

```diff
@@ -374,7 +374,9 @@ export function getContainerStyle({ id, tag, CONTEXT, CLASS, ANIMATION } : { id
         }
 
         #${ id }.${ tag }-context-${ CONTEXT.IFRAME } .${ CLASS.OUTLET } iframe {
-            width: 100%;
+            width: 1px;
```

Comment:
Not sure I understand the fix -- could you explain the 1px and the *width?

Reply:
The close button is clipped on iphone 5 portrait mode
![iphone 5 ios 10 3 14e8301 2017-10-29 17-05-26](https://user-images.githubusercontent.com/3213579/32150125-6ca8ba46-bccb-11e7-9f48-b09a638b79eb.png)
more details here https://stackoverflow.com/questions/23083462/how-to-get-an-iframe-to-be-responsive-in-ios-safari?answertab=votes#tab-top

Reply:
Makes sense! I think we can drop the *width here though since we only need to support IE9+